--- ladspa-0.4.16/Makefile.am.orig	2016-09-27 05:48:53.144159402 +0200
+++ ladspa-0.4.16/Makefile.am	2016-09-27 05:49:55.017492041 +0200
@@ -34,7 +34,7 @@
 	latency_1914.la xfade_1915.la sc4m_1916.la \
 	mbeq_1197.la pitch_scale_1193.la pitch_scale_1194.la imp_1199.la
 
-SUBDIRS = po util gsm gverb metadata
+SUBDIRS = po util gverb metadata
 
 # Wacky stuff to stop automake getting confused
 EXTRA_DIST = config.rpath  @top_srcdir@/*.xml @top_srcdir@/*.c @top_srcdir@/*.h \
@@ -73,7 +73,7 @@
 sc4_1882_la_LIBADD = -Lutil -ldb -lrms
 sc4m_1916_la_LIBADD = -Lutil -ldb -lrms
 se4_1883_la_LIBADD = -Lutil -ldb -lrms
-gsm_1215_la_LIBADD = gsm/libgsm.a
+gsm_1215_la_LIBADD = -lgsm
 gverb_1216_la_LIBADD = -Lgverb -lgverb
 lcr_delay_1436_la_DEPENDENCIES = util/biquad.h
 
