--- ladspa-0.4.17/configure.ac.orig	2019-04-09 19:18:11.919891611 +0200
+++ ladspa-0.4.17/configure.ac	2019-04-09 19:20:51.815692048 +0200
@@ -102,7 +102,7 @@
 else
   AC_MSG_WARN([Can't find architecture specifaction in CFLAGS])
 
-  CFLAGS="$CFLAGS -Wall -O3 -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -fPIC -DPIC ${USE_SSE} ${DARWIN_CFLAGS}"
+  CFLAGS="$CFLAGS -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -fPIC -DPIC ${USE_SSE} ${DARWIN_CFLAGS}"
 
   dnl For Intel's C compiler use:
   dnl CC="icc"
